### PR TITLE
Added RemoteView command.

### PR DESCRIPTION
**Pull request by ghost**
Took a while to understand how this command builder thing works but eh at least I finished it.

### CODE REVIEW

**Comment by ghost** [comment]
So this makes your POV be from another entity?

**Comment by ghost** [comment]
Yep.

EDIT:
      `Could not resolve com.mojang:patchy:1.2.3.`
         `Could not get resource 'https://maven.terraformersmc.com/com/mojang/patchy/1.2.3/patchy-1.2.3.pom'.`
            ` Could not GET 'https://maven.terraformersmc.com/com/mojang/patchy/1.2.3/patchy-1.2.3.pom'. Received status code 521 from server: Origin Down`

oof

**Comment by ENDER1355** [comment]
> Took a while to understand how this command builder thing works but eh at least I finished it.

The command was bugged in b73

File: src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/CommandRemoteView.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                            .begin<String>("name")
+                            .verifiedBy(ParameterBuilder.STRING_VALIDATOR)
+                            .useMinecraftAutoCompletion()
+                            .required()
+                            .build()
+                    )
+                    .handler { command, args ->
+                        val name = args[0] as String
+                        for (entity in mc.world!!.entities) {
+                            if (name == entity.entityName.lowercase()) {
```
Comment (1zun4secondary) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/CommandRemoteView.kt]: Use: "example".equals("EXAMPLE", ignoreCase = true)

Comment (ghost) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/CommandRemoteView.kt]: Done. `lowercase()` alone looked better though but if it works, then no change is needed.

**Review by 1zun4secondary — APPROVED** [review]
Looks good.

**Pull request merged**